### PR TITLE
fixed alignment of icon in alert.component

### PR DESCRIPTION
**Pull request by martinkrol-poi**
added settings for carcat.scss regarding alert.component

### CODE REVIEW

File: projects/material-addons/src/lib/alert/alert-configuration.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+export interface AlertConfiguration {
+  type?: AlertType;
+  size?: AlertSize;
+}
+
+export const MAD_ALERT_CONFIGURATION = new InjectionToken<AlertConfiguration>('mad-alert-configuration');
+
+export const MAD_ALERT_CONFIGURATION_PROVIDER = {
+  provide: MAD_ALERT_CONFIGURATION,
+  deps: [[new Optional(), new SkipSelf(), new Inject(MAD_ALERT_CONFIGURATION)]],
```
Comment (moritzpatterer) [inline projects/material-addons/src/lib/alert/alert-configuration.ts:13]: the factory should not have any dependencies as this should be the MAD default options (so the default default if you will)

File: projects/material-addons/src/lib/alert/alert-configuration.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { Inject, InjectionToken, Optional, SkipSelf } from '@angular/core';
+import { AlertType, AlertSize } from './alert.component';
+
+export interface AlertConfiguration {
+  type?: AlertType;
+  size?: AlertSize;
+}
+
+export const MAD_ALERT_CONFIGURATION = new InjectionToken<AlertConfiguration>('mad-alert-configuration');
```
[the remark this answers is not in the document]
Comment (moritzpatterer) [reply projects/material-addons/src/lib/alert/alert-configuration.ts:9]: further I'd suggest writing it similar to how Material Components do they're default settings (example pasted below)

--

/** Injection token to be used to override the default options for `mat-menu`. */
export const MAT_MENU_DEFAULT_OPTIONS = new InjectionToken<MatMenuDefaultOptions>(
  'mat-menu-default-options',
  {
    providedIn: 'root',
    factory: MAT_MENU_DEFAULT_OPTIONS_FACTORY,
  },
);

/** @docs-private */
export function MAT_MENU_DEFAULT_OPTIONS_FACTORY(): MatMenuDefaultOptions {
  return {
    overlapTrigger: false,
    xPosition: 'after',
    yPosition: 'below',
    backdropClass: 'cdk-overlay-transparent-backdrop',
  };
}

**Review by moritzpatterer — CHANGES_REQUESTED** [review]

File: projects/material-addons/src/lib/alert/alert-configuration.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (martinkrol-poi) [reply projects/material-addons/src/lib/alert/alert-configuration.ts]: done

File: projects/material-addons/src/lib/alert/alert-configuration.ts (hunk 1 above)
Comment (martinkrol-poi) [reply projects/material-addons/src/lib/alert/alert-configuration.ts]: done

**Review by moritzpatterer — APPROVED** [review]

**Pull request merged**